website/public/locale/en/translation.yaml

```diff
@@ -115,21 +120,21 @@ faq:
-    question2: What are the minimum requirements? What are the supported container orchestrators?
-    question3: Why would you use OpenEBS on EBS?
-    question4: Why did you choose iSCSI? Doesn’t it introduce latency and decrease performance?
-    answer4: With OpenEBS, we strive to make it simple to use - using Kubernetes as much as possible to manage OpenEBS itself. iSCSI allows us to be more resilient in cases in which the workload and the controller are not on the same host. In other words, the OpenEBS user or architect will not suffer an outage in the case that the storage IO controller is not scheduled local to the workload in need of storage. OpenEBS does a variety of things to improve performance elsewhere in the stack - and much more is to come via the cStor storage engine - to be able to have this level of flexibility.
-    question5: How does OpenEBS provide higher availability for stateful workloads?
-    question6: What changes are needed to Kubernetes or to other subsystems to leverage OpenEBS?
-    question7: How do you get started? What is a typical trial deployment?
-    question8: What is the most common use case? What are the most common workloads?
-    answer8: Any stateful workload running on Kubernetes likely has been run on OpenEBS at least once. Typical workloads include Prometheus and FluentD and other stateful workloads used to manage Kubernetes itself. Jenkins, all flavor of database including even in some cases NoSQL databases, and object storage are all widely deployed.
-    question9: Where is my data stored? How can I see that?
-    question10: How about performance? How does OpenEBS perform?
-    question11: How is the data protected? Please explain what happens when a host fails, or a client workload fails or a data center fails?
-    answer11: Kubernetes, of course, gives many ways to enable resilience. We leverage these wherever possible. For example, let’s say the IO container that has our iSCSI target fails. Well, it is spun back up by Kubernetes. The same applies to the underlying replica containers, where the data is stored. They are spun back up by Kubernetes. Now - the point of replicas is to make sure that when one or more of these replicas is being respun and then repopulated in the background by OpenEBS, the client applications still run. OpenEBS takes a simple approach to ensuring that multiple replicas can be accessed by an IO controller using a configurable quorum or the minimum number of replica requirements. Besides, our new cStor checks for silent data corruption and in some cases, can fix it in the background. Silent data corruption, unfortunately, can occur from poorly engineered hardware and from other underlying conditions, including those that your cloud provider is unlikely to report or identify.
+  question1: What is most distinctive about OpenEBS architecture?
```

Comment:
Why these changes ?

Reply:
Did the format document whole file got formatted